api/v1alpha1/etcdcluster_webhook.go

```diff
@@ -106,3 +121,66 @@ func (r *EtcdCluster) ValidateDelete() (admission.Warnings, error) {
 	etcdclusterlog.Info("validate delete", "name", r.Name)
 	return nil, nil
 }
+
+// validatePdb validates PDB fields
+func (r *EtcdCluster) validatePdb() (admission.Warnings, field.ErrorList) {
+	if r.Spec.PodDisruptionBudget == nil {
+		return nil, nil
+	}
+	pdb := r.Spec.PodDisruptionBudget
+	var warnings admission.Warnings
+	var allErrors field.ErrorList
+	if pdb.Spec.MinAvailable != nil && pdb.Spec.MaxUnavailable != nil {
+		allErrors = append(allErrors, field.Invalid(
+			field.NewPath("spec", "podDisruptionBudget", "minAvailable"),
+			pdb.Spec.MinAvailable.IntValue(),
+			"minAvailable is mutually exclusive with maxUnavailable"),
+		)
+	} else {
+		minQuorumSize := r.CalculateQuorumSize()
+		if pdb.Spec.MinAvailable != nil {
```

Comment:
MinAvailable can contain value such as `50%`. Looks like it won't pass validaition here